source/styles/main.css

```diff
@@ -11,6 +25,38 @@ nav a:hover {
   cursor: pointer;
 }
 
+.navbar {
+  border-bottom: black solid 2px;
+}
+
+.navbar-brand {
+  background: none;
+  border: none;
+  padding: 0;
+}
+
+#explore-nav {
```

Comment:
try to use classes (like .class-name) in CSS instead of ids. you can add additional classes to each button for the styles that aren't common between them if you have to. for example, if you style .nav-link for the background and border, we will have less duplicate code.
```
.nav-link {
  background: none;
  border: none;
  color: black;
  margin-left: 5rem;
}
```

Reply:
Fixed!